### DISCUSSION TITLE
Add a button to skip current generation please!

### DISCUSSION THREAD

**Discussion by ZeroCool22** [MoonRide303/Fooocus-MRE#162] [Ideas]
![Screenshot_19](https://github.com/MoonRide303/Fooocus-MRE/assets/13344308/d6b1e454-a017-4949-80f9-96b52331a183)

Sometimes watching the preview, we already know that it's not the image we want, so would be nice to have a **Skip button**.

Thx!

**Comment by spyderr7** [comment]
Yes! We need skip button!
amazing job MoonRide btw!

**Reply by barepixels** [reply]
VOTE

**Comment by Sirmadsen** [comment]
yes pls. Fooocus already have this, don't understand the need to remove it.
